blivet/devicelibs/mdraid.py

```diff
@@ -343,6 +343,9 @@ def mddetail(device):
 
     info = {}
     for (name, colon, value) in (line.strip().partition(":") for line in lines):
+        if name and name[0] not in string.letters:
+            continue
```

Comment:
Looks good to me. I don't we are interested in any other values than those separated by ``" : "``.